packages/integration-tests/src/components/events/test-clipboard-event-composed/clipboard-event-composed.spec.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+const assert = require('assert');
+const URL = '/clipboard-event-composed';
+
+// https://stackoverflow.com/questions/11750447/performing-a-copy-and-paste-with-selenium-2/41046276#41046276
+function copy() {
+    browser.keys(['Control', 'Insert', 'Control']);
+}
+function cut() {
+    browser.keys(['Control', 'Delete', 'Control']);
+}
+function paste() {
+    browser.keys(['Shift', 'Insert', 'Shift']);
+}
+
+describe('clipboard-event-composed polyfill', () => {
+    beforeEach(() => {
+        browser.url(URL);
+    });
+
+    // This test suite relies on the Selection API which is not supported in ie11
+    if (browser.config.capabilities.commonName !== 'ie11') {
+        it('copy event should be composed', () => {
+            browser.$(function() {
+                window.getSelection().selectAllChildren(document.body);
+            });
+
+            copy();
+
+            const didHandleCopy = browser.execute(() => {
+                var container = document.querySelector('integration-clipboard-event-composed');
+                return container.didHandleCopy();
+            });
+
+            assert.strictEqual(didHandleCopy, true);
+        });
+
+        // The cut() function is not working as expected.
```

Comment:
Can you add more details on why the cut method doesn't work as expected? Is it true for all browsers?

Reply:
I have no idea why it doesn't work. We should have sufficient coverage via `copy` and `paste` so I removed it.